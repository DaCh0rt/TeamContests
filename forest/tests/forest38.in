516 594
-1769 -2145
-4525 -2637
628 5578
5396 -5583
8101 9660
-5941 8776
-8625 9166
-8885 3259
-8283 8189
7002 -13
7580 -9652
-8185 4948
-6236 744
6662 -392
-2327 5366
870 2052
-3244 -7478
3560 -285
-3677 -7004
-2205 -2717
-3619 5374
4986 -8152
-8139 -5365
6510 -5004
-4753 5883
-9865 1907
2255 -6965
-1707 4399
-6523 3675
-6906 -6798
-6861 -2154
-3498 -7132
3615 6816
-6929 4617
5286 -428
-5165 -5636
2502 6641
-9432 -2793
-8103 3160
-2856 717
-9435 -793
-4576 -1267
3690 6786
-5218 4347
8470 5000
-3292 1050
-7931 3543
5185 -9435
-4487 -4652
-1027 -4107
1337 7452
780 -1138
-8102 -4639
2732 3829
-8005 1938
5858 -867
-9914 3103
7339 -6844
8321 429
1699 6363
-6803 3147
-4589 4423
-8322 -7513
-4796 -2066
8764 283
1999 -119
-4256 -1742
-3301 -6238
919 3108
6191 -5229
3671 -882
-5338 8826
-3364 -7047
1866 -3345
-4474 -4208
9905 -4987
-8340 -5913
4826 6013
-8166 1323
882 -6149
6655 2402
6666 124
9053 -2085
6113 1255
251 8596
-1956 -829
2038 -5994
-7975 9732
3314 -2150
-1475 -5837
-2713 -2966
88 8329
3095 325
-7643 3205
3529 -4243
5155 -2566
-3485 1402
2325 391
3108 -1197
8753 -3755
1868 -4631
3714 -4309
-1770 7928
-3304 7967
703 5266
345 1715
7077 -9513
-9699 -6161
871 -3921
7858 6742
-3705 -8727
-4268 3431
7798 -2056
3825 8457
3238 3381
6190 -3437
8901 1567
-1641 -54
-2862 530
4953 4889
1577 1338
-3381 9647
8535 7463
-3130 816
-3231 -1796
9708 3326
997 -3260
-5513 -6953
-3017 -5336
-8835 -311
1357 2952
-3539 7256
4363 -5465
6294 -5824
8440 -1833
1086 -4276
6102 -9192
3208 3481
-37 -6522
-1560 -8886
-8219 -677
1275 -2510
3357 -2732
3013 -2465
7794 -3363
3630 -6587
-9186 2595
2257 -8587
205 -8526
9143 -7752
1202 9128
-3393 -3090
5826 -6885
1113 4300
-5788 3274
1300 -1911
-6290 2440
806 -1953
-4372 -6096
5172 4497
-9632 -6575
-422 759
7644 -4631
-1039 9857
4825 5718
876 -9985
74 -4583
4237 8842
1527 8296
-6971 8496
-3057 5025
-6027 8874
-5386 9254
2929 9914
8602 6676
-5760 -7245
-8907 4818
-7838 -4332
-5734 8085
-4769 -5067
9282 -3817
6227 -6591
-4016 2832
-3610 5835
4098 823
4067 -2728
-2089 3397
-7363 9469
4201 6275
-8968 -9298
3174 -5556
334 5681
-2835 -7175
-6174 2334
6826 2715
4447 8525
-2 7786
-6723 2301
-6379 -5118
-430 3154
1012 5752
1232 3150
8380 4499
4289 2283
8488 -2698
-4268 -7665
-7072 6078
9636 -7903
-8853 -5768
4421 5636
-2551 -3849
-9187 4073
7094 -9898
9352 -6002
6256 -8676
-2920 -6166
9454 3182
-8832 3629
4871 -7682
8070 9641
-9390 6691
5730 -2621
3884 3715
1401 -2185
-4896 -8533
-7790 7333
-6338 7231
3351 6360
9329 3384
-7974 -7044
-4707 1315
2836 5072
-1905 311
-3550 -1718
2793 4345
8086 -6316
334 -5470
-477 56
-6518 6621
6571 1339
7760 3354
-1842 -8317
4929 5464
-5517 4176
5658 -4257
9721 -7737
6300 2631
430 -8614
2993 -3648
430 -4515
-615 5961
1884 3660
-7157 -9976
-774 -5210
2466 -500
-7759 -2662
-6224 -8830
-6371 -1406
4777 5458
8241 -4866
-7715 2647
4610 -1345
-7864 -9072
-3597 -703
9797 -589
-5076 9908
-5456 8741
837 4232
1752 3102
-7730 -556
7333 6981
4718 -1076
-9996 6302
5245 7111
-3461 1250
1423 -7733
-7130 1627
7476 2186
4818 -9709
-5214 9314
693 -9895
-778 6641
7363 -8101
7960 6436
-2225 -2139
-4856 1756
2686 9733
-3630 8038
18 5545
-1134 -453
-5956 -65
2156 -3693
1217 -6959
8783 6463
-5264 3926
114 -2619
-2389 6418
-9296 -163
6580 -9438
7941 2652
-7595 2054
4229 7288
-2553 -1388
7222 2558
8706 3777
5311 -4416
6221 1961
4568 -4963
4816 -6362
6373 6590
-5661 8884
8354 200
4354 6561
-3848 -1340
5082 -1514
-1973 -2196
-5566 -3431
897 -6460
3343 -3938
3377 2951
-204 -9379
-6141 7328
9012 -6607
9445 -3261
-251 8793
4215 1202
3232 1847
9294 -5171
-8996 -7981
3071 -3880
5369 5768
-1139 -1213
-6304 6032
9852 7790
-4867 -9468
-5683 -4136
-4116 -2799
-1681 6837
4824 735
5143 9094
-3062 -1889
608 125
-7674 4756
-5752 -9768
-6465 -711
9152 3979
-2570 -7130
-6047 -4706
1018 4635
3812 -1778
3538 -1489
548 1151
7443 -9675
2823 -9395
-413 8960
-7877 -7550
-9640 -2848
-9835 2547
-4574 1085
4321 3024
7938 7613
-1242 -507
2173 6448
8707 7295
191 -9251
-7835 996
4528 8539
3158 3528
-5773 -1830
-140 -339
-2057 -976
9684 2586
4008 -5259
-1252 -3000
6886 6869
1131 -4244
8442 -3696
-427 -3631
9327 9138
9092 -3249
9735 -4286
-4472 -1584
8681 3997
-3588 -368
-5277 5708
7198 8658
-502 6249
-3144 -21
3069 7581
-397 -2267
-6917 -5799
5774 3929
6702 723
228 4168
-3048 -2481
-8289 663
7984 248
1962 4390
6506 7689
4500 9152
-9266 6543
-6822 4193
4250 -2467
57 -4606
-3655 -8220
7742 7867
-6783 2225
6793 -346
-1004 -704
-2186 8947
-5719 1572
-280 6317
4872 -3616
2216 -1674
-260 -1335
4230 5819
-748 -4376
-5841 804
2051 -4156
-7413 9753
-2306 4847
-6871 -6893
-2095 8905
-6578 7482
9202 -6989
-1840 -830
6290 -3567
-1170 4621
7639 1262
-100 2543
4663 6824
-7718 4500
4175 7826
9798 -3920
-6389 3095
-430 -4551
1549 -2011
8764 -7742
3147 1003
-395 3736
-2546 9750
-4658 -4466
704 -7370
8201 -191
-5330 7819
-1205 -6670
-2879 -463
52 -277
-6195 8719
4048 -4845
7600 -913
2908 2264
-9182 -7891
-738 7364
4550 -2998
60 -4667
6568 -173
588 40
5515 -5652
1410 8532
682 -9037
9575 916
-2561 3469
7216 -474
-1132 -9753
-3997 -5253
258 -6088
2703 -3177
9433 -586
9309 1909
8680 -6443
-9713 -8810
-1004 4251
-1710 -2429
2224 -6057
9806 -1050
-5951 1006
2583 -9392
-9947 9569
-9250 5096
9415 -9975
520 -3226
-3363 4892
6547 2675
-8160 9251
7767 -2894
2844 -5960
-7458 5407
4887 4610
-4590 -5211
-7591 -8473
8438 8024
-2448 -4164
1255 -1686
-3097 -8315
1251 6126
-1287 7566
-2709 5812
2613 4298
-5183 4018
4771 1762
-170 7804
-1254 2730
-9533 9252
-5779 8132
3305 9184
-9936 122
1779 1769
-6391 7619
-3872 -4506
6242 1261
-7798 3972
3364 -831
-5898 -843
-6297 -6535
-4505 973
