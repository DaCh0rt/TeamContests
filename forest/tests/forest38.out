80148.53 434631485.72
